pages/linux/sslstrip.md

```diff
@@ -0,0 +1,20 @@
+# sslstrip
+
+> Perform Moxie Marlinspike's SSL stripping attacks with this MITM tool.
```

Comment:
```suggestion
> Perform Moxie Marlinspike's Secure Sockets Layer (SSL) stripping attacks with this Man-in-the-Middle (MITM) tool.
```
For technical terms, we generally add abbreviations too for more beginner-friendliness for those who might not be aware of the terms. Feel free to reword my suggestion.